lightning/pkg/importer/table_import.go

```diff
@@ -262,6 +264,26 @@ func (tr *TableImporter) importTable(
 		}
 	}
 
+	// turn on/off "force_partition_range" for table
+	if isLocalBackend(rc.cfg) {
```

Comment:
Why here still isLocalBackend? for global sort backend is not local backend. how can we handle the externbackend?